build/build.sh

```diff
@@ -257,7 +257,7 @@ run_test_cover() {
     pre_build
     pushd $SrcPath >/dev/null
     echo -n "${TPATH}"
-    go test -trimpath -covermode=count --coverprofile coverage.txt $(go list ./... | grep -v depends)
+    go test -trimpath -covermode=count --coverprofile coverage.txt $(go list ./... | grep -v depends | grep -v master)
```

Comment:
why not include master?

Reply:
> why not include master?

currently  not include and resove it later @baijiaruo